docs/content/guides/cell-functions/cell-renderer.md

```diff
@@ -36,82 +37,238 @@ validator: Handsontable.NumericValidator,
 type: 'numeric'
 ```
 :::
+:::
 
```

Comment:
```suggestion
::: only-for react
## Overview

A renderer is a function that determines how a cell looks.

Set together, a renderer, [editor](@/guides/cell-functions/cell-editor.md) and [validator](@/guides/cell-functions/cell-validator.md) form a [cell type](@/guides/cell-types/cell-type.md).
:::
```

I know our changes are just a base for further content improvements, but we need some sort of introduction here.